raceforcash/src/components/registration/registration.component.tsx

```diff
@@ -1,9 +1,71 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import { Form, Container, Row, Col, Button } from 'react-bootstrap';
 
 export const RegistrationComponent: React.FC = () => {
+
+    const [inputEmail, setInputEmail] = useState('');
+    const [inputUsername, setInputUsername] = useState('');
+    const [inputPassword, setInputPassword] = useState('');
+    const [inputFirstName, setInputFirstName] = useState('');
+    const [inputLastName, setInputLastName] = useState('');
+
+    useEffect(() => {
+      }, []);
+
+    //   const loginCredentials = async () => {
+        const loginCredentials = () => {
```

Comment:
is this for logging in or registering? if they are login in then we just need username and password, if they are registering we need more info here